hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/snapshot/SnapshotDiffHandler.java

```diff
@@ -63,7 +63,10 @@ public class SnapshotDiffHandler extends Handler {
 
   @CommandLine.Option(names = {"-p", "--page-size"},
       description = "number of diff entries to be returned in the response" +
-          " (optional)")
+          " (optional)",
```

Comment:
nit:
1. remove `(optional)` similar to length in [ListOptions](https://github.com/apache/ozone/blob/f9ec90c4002ddb68d3cb89e88c0d7c48d99979e3/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/ListOptions.java#L32).
2. 1000 is also maxed page size allowed [here](https://github.com/apache/ozone/blob/5dcdf85c7aca22cea8f39a64888e6519f1a5b2bb/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java#L740), we should add this in the description.

Reply:
Do you mean to change it like this:
`description = "number of diff entries to be returned in the response, as defined by" + " ozone.om.snapshot.diff.max.page.size",`

Reply:
Just remove the `(optional)` for now. I'll think about the second point and create a follow-up task if needed.

Reply:
Submitted, could you please review it again @hemantk-12